pages/common/fdroid.md

```diff
@@ -0,0 +1,29 @@
+# fdroid
+
+> F-Droid build tool.
+> F-Droid is an installable catalogue of FOSS (Free and Open Source Software) applications for the Android platform.
+> More information: <https://f-droid.org/>.
+
+- Build a specific app:
+
+`fdroid build {{app_id}}`
+
+- Build a specific app in a build server VM:
+
+`fdroid build {{app_id}} --server`
+
+- Publish the app to the local repository:
+
+`fdroid publish {{app_id}}`
+
+- Install the app on a connected device:
```

Comment:
FDroid automatically determines to which connected device the app will be installed?

Reply:
If several devices are available, it will install the app on all of them: https://gitlab.com/fdroid/fdroidserver/blob/c74dc730dd50c052a2233bd165855e52718b7972/fdroidserver/install.py#L101

Reply:
I think it would be cool to have that stated. Would you mind doing that?

Reply:
I changed the explanation. Is that better?

Reply:
Looks great to me. 😄